package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "channelape-web-service-sdk",
-	"version": "0.1.0",
+	"version": "0.1.0-develop.0",
```

Comment:
0.1.0 has already been released. Please update this version to "0.2.0-develop.0"